pkg/cmd/flagtypes/addr.go

```diff
@@ -55,6 +55,16 @@ func (a *Addr) String() string {
 	return a.URL.String()
 }
 
+// HostPort returns the host and port joined together. If no port is set :0 will
```

Comment:
Seems like an odd default.  It didn't used ot choose 80 or 443 by default?

Reply:
There's a default port field already - if you didn't set that, basically we throw our hands up and say "you are stupid and we want you to fail".